src/main/java/frc/robot/RobotContainer.java

```diff
@@ -35,6 +36,14 @@ public class RobotContainer {
   private final GripperSystem gripperSystem;
   private final XboxController driver = new XboxController(OperatorConstants.kDriverControllerPort);
   private final JoystickButton xButton = new JoystickButton(driver, XboxController.Button.kX.value);
+  
+  private final Joystick joystick;
```

Comment:
joystick variable should indicate which one it is, driver left or right?
Button names should also include the function of the button in the name